source/conf.py

```diff
@@ -513,8 +513,12 @@ def customReplacements(app, docname, source):
     "|CHECKSUMS_URL|" : "https://packages.wazuh.com/4.x/checksums/wazuh/",
     "|RPM_AGENT|" : "https://packages.wazuh.com/4.x/yum/wazuh-agent",
     "|RPM_MANAGER|" : "https://packages.wazuh.com/4.x/yum/wazuh-manager",
+    "|RPM_AGENT_DEBUG_SYMBOLS|" : "https://packages.wazuh.com/4.x/yum/wazuh-agent-debuginfo",
+    "|RPM_MANAGER_DEBUG_SYMBOLS|" : "https://packages.wazuh.com/4.x/yum/wazuh-manager-debuginfo",
     "|DEB_AGENT|" : "https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-agent/wazuh-agent",
     "|DEB_MANAGER|" : "https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-manager/wazuh-manager",
+    "|DEB_MANAGER_DEBUG_SYMBOLS|" : "https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-manager/wazuh-manager-dbg",
+    "|DEB_AGENT_DEBUG_SYMBOLS|" : "https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-agent/wazuh-agent-dbg",
```

Comment:
This URL should be checked. I'm not a specialist, but IMHO debuginfo packages should be located in the same folder of the wazuh package